src/Microsoft.ML.FastTree/Microsoft.ML.FastTree.csproj

```diff
@@ -27,6 +32,8 @@
     <Compile Include="FastTreeRegression.cs" />
     <Compile Include="FastTree.cs" />
     <Compile Include="FastTreeTweedie.cs" />
+    <Compile Include="GamClassification.cs" />
+    <Compile Include="GamRegression.cs" />
     <Compile Include="GamTrainer.cs" />
```

Comment:
Actually this is somewhat peculiar. I know you didn't do this, but why is the explicit listing in this project necessary at all? Does anything break if you get rid of this entire `ItemGroup` with all the `<Compile Includes`?